shotgun_api3/shotgun.py

```diff
@@ -66,19 +66,20 @@
 
 SG_TIMEZONE = SgTimezone()
 
-
 NO_SSL_VALIDATION = False
 try:
-    import ssl
-    if os.environ.get("SHOTGUN_DISABLE_SSL_VALIDATION", False):
+    import ssl        
+except ImportError, e:
+    if os.environ.get("SHOTGUN_FORCE_CERTIFICATE_VALIDATION") not in [0, "0", "", None]:
```

Comment:
This form of checking the value of the env var is feeling pretty ugly.  I think

```python
if "SHOTGUN_FORCE_CERTIFICATE_VALIDATION" in os.environ:
   ...
```

Would be simpler and we can just say "if the env var is set" rather than worrying about a particular value.